change/office-ui-fabric-react-2020-10-06-08-02-45-feat-variants-cherrypick.json

```diff
@@ -0,0 +1,8 @@
+{
+  "type": "patch",
```

Comment:
unexpected change file. remove

Reply:
most of the change files look wrong; could you remove all and re-generate with accurate comment

Reply:
will do, good catch.